frontend/templates/ng-templates/browser-region.html

```diff
@@ -8,19 +8,20 @@
     <div class="col-md-12">
         <h1 ng-if="ctrl.rsid">{{ ctrl.rsid }}</h1>
         <h1 ng-if="!ctrl.rsid">Region: {{ ctrl.region.chrom }}
+	    <span ng-if="ctrl.region.statusCode">{{ ctrl.region.variantId }}</span>
```

Comment:
Why checking statusCode here: `ng-if="ctrl.region.statusCode"`?

Reply:
I think my original idea was to show the variant in case of bad load, but it doesn't really make sense as it's anyway visible in the URL